### DISCUSSION TITLE
what is i18n edge package?

### DISCUSSION THREAD

**Discussion by lonagi** [nuxt-modules/i18n#2202] [General]
What is difference between simple i18n and i18n edge packages?

**Comment by paulvonber** [comment]
i18n edge has more bugs...

**Comment by kazupon** [comment]
Same as Nuxt3 and Edge Release Channel.
Nuxt i18n module is undergoing commits, improvements and bug fixes.
You can opt-in to be the first to test it before the next release.

See https://github.com/nuxt-modules/i18n#edge-release-channel
